client/post-editor/editor-title/index.jsx

```diff
@@ -39,6 +42,8 @@ const EditorTitle = React.createClass( {
 
 	getDefaultProps() {
 		return {
+			siteId: 0,
```

Comment:
So it's not any valid siteId? 
You are for `null` ? I always have trouble deciding... :/